fhirAssembly/README.md

```diff
@@ -0,0 +1,298 @@
+[FHIR server](https://hl7.org/fhir/R4/index.html).  FHIR (Fast Healthcare Interoperability Resources) is
+a healthcare information exchange standard that makes use of an HL7-defined set of _resources_
+to support information sharing in a number of ways, including, but not limited to, including documents, 
+messages, services, and RESTful interfaces.
+
+The assembly created in this project allows a Vantiq user to use the service defined herein to communicate with the 
+FHIR server. This is done using the service and associated procedures that are closely tied to FHIR _interactions_.
+We assume that the user is familiar with FHIR resources and the interaction interfaces, as the Objects used are the 
+native FHIR types presented in a Vail-friendly manner.
+
+This project does NOT provide a FHIR server, nor any new FHIR resource definitions. FHIR is an interoperability 
+definition, and there is no point in trying to define another on top of it.  What this project defines is a Vantiq 
+Service that provides for the FHIR _interactions_ with that FHIR server in a Vantiq application.
+
+## Prerequisites
+
+Use of this assembly presumes the following
+
+* Vantiq
+    * The user is familiar with the Vantiq development environment.
+    * The user is familiar with installing an assembly.
+    * The user as access to a Vantiq server and namespace into which to install the assembly and perform further
+      development.
+* FHIR
+    * The user is familiar with FHIR resources and services.
+    * The user has access to a FHIR server.
+        * The type of access defined will determine what interactions the user can perform.  This project does not 
+          attempt to determine that or limit it in any way.
+
+## The FHIR Connection Assembly Structure
+
+The FHIR Connection assembly defines a number of Vantiq components used for interacting with the FHIR server.  We 
+outline those defined below. These are all found in the package `com.vantiq.fhir`.
+
+### Types
+
+* `FHIRResponse` -- This is the response type from FHIR interactions.
+    * `statusCode` -- (Integer) The HTTP status code returned by the underlying REST operation
+    * `headers` -- (Object) The HTTP headers returned by the underlying REST operations.  The object returned 
+      consists of properties whose name is the header name, and whose value is the header value.
+    * `body` -- (Object) The Vantiq Object representation of the FHIR resource returned by the FHIR interaction.  
+      Depending upon the interaction, this might be a FHIR resource requested (_e.g._, the _created_
+      [Encounter](https://hl7.org/fhir/R4/encounter.html)) or information/errors returned in the form of an 
+      [OperationOutcome](https://hl7.org/fhir/R4/operationoutcome.html).
+    * These data items are often important to the interpretation of an interaction's outcome.
+* `Modifiers` -- This can be used as a general parameter to most operations. It generally contains , and 
```

Comment:
Incomplete thought: `It generally contains , and`

Reply:
Thanks.